cmd/internal/sleep.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+// dataGatherCmd represents the dataGather command
+var sleepCommand = &cobra.Command{
+	Use:   "sleep",
+	Short: "Sleep forever",
+	Long: `Do nothing forever.
+
+This is used as a dummy command for the volume-management-pods.
+
+`,
+	RunE: func(cmd *cobra.Command, args []string) (err error) {
+		return Sleep()
+	},
+}
+
+// Sleep will sleep indefinitely
```

Comment:
It sleeps 24 hours.

Reply:
Inside an endless loop :wink:

Reply:
I see the comment was on the function :)